stories/Molecules/Blocks/StatsCards/statscards.scss

```diff
@@ -0,0 +1,217 @@
+@import '../../../assets/scss/breakpoints';
+@import '../../../assets/scss/variables';
+
+/* statscard start */
+.statscard {
+  background-color: $color-gray-200;
+
+  & span {
+    display: block;
+  }
+
+  &__title {
+    color: $color-black;
+    font-family: $Font-Family-ProximaBold;
+    letter-spacing: 0;
+    margin-bottom: $spacing-40;
+
+    @include devicebreak(medium) {
+      margin-bottom: $spacing-80;
+    }
+  }
+
+  &__number {
+    color: $color-gray-200;
+    font-family: $Font-Family-SohneBold;
+    font-size: 80px;
```

Comment:
@Shubham-Sharma-77 going forward can we use the font size in rem and letter spacings also